### DISCUSSION TITLE
Need help to config the pnp search vertical please...It has been setup and looks ok but does not work...

### DISCUSSION THREAD

**Discussion by steveh2023** [microsoft-search/pnp-modern-search#3761] [Q&A]
Hi,

A pnp search vertical has been setup and it all looks ok but it just does not work, so we have missed something and need help please.

We are trying to filter the search results based on various values one of which is: "Division C" This value is in a SharePoint document library column called "Owner division" internal name = ows_Ownerdivision - a mapping to RefinableString11 has been done in the search schema with an alias of RGT-Merlin-OwnerDivision...

The tab value for "Division C" is currently defined as RGT-MERLIN-OWNERDIVISION:Division C

The results are from the same document library this column is in and a reindex has been requested more than 24 hrs ago after config.

However when the vertical for Division c is clicked the search results go to zero but there are hundreds of documents tagged with the "Division C" value in the document library.

Here are some screenshots explaining the issue in more detail:

https://github.com/microsoft-search/pnp-modern-search/assets/144025343/8706f050-a7fb-424c-9a6a-4bdc05520c7c

This appears to be setup as per documentation but something is missing as it does not work...

Does anyone have any suggestions of alternative configs please?

**Comment by kasperbolarsen** [comment]
First step should be to verify your data. Try the queries in SP Editor or SharePoint Search Query tool. 
This will both show if the docs have been indexed, and if the mappings are in effect
